### DISCUSSION TITLE
Invalid Links

### DISCUSSION THREAD

**Discussion by Lemodo20** [abuzuhri/Amazon-SP-API-CSharp#411] [General]
Hi,

the Links at your Website

https://github.com/abuzuhri/Amazon-SP-API-CSharp

Order List, For more orders sample please check [Here](https://github.com/abuzuhri/Amazon-SP-API-CSharp/blob/main/Source/FikaAmazonAPI.Test/Reports.cs)

Report List, For more report sample please check [Here](https://github.com/abuzuhri/Amazon-SP-API-CSharp/blob/main/Source/FikaAmazonAPI.Test/Orders.cs)

Product Pricing, For more Pricing sample please check [Here](https://github.com/abuzuhri/Amazon-SP-API-CSharp/blob/main/Source/FikaAmazonAPI.Test/ProductPricing.cs)

and follwing don't work 

So can't check your examples :-(

**Comment by ProNotion** [comment]
[Check the samples in the project source](https://github.com/abuzuhri/Amazon-SP-API-CSharp/tree/main/Source/FikaAmazonAPI.SampleCode)

**Reply by abuzuhri** [reply]
all link now fixed
